package.json

```diff
@@ -322,7 +326,8 @@
     "@diplodoc/quote-link-extension": "^0.1.1",
     "@diplodoc/tabs-extension": "^3.5.1",
     "@diplodoc/transform": "^4.43.0",
-    "@gravity-ui/uikit": "^7.1.0",
+    "@gravity-ui/components": "^4.2.2",
+    "@gravity-ui/uikit": "^7.10.0",
```

Comment:
please don't raise peerDeps uikit. The project using the editor will have a later version, but we must maintain backward compatibility

Reply:
In this case it's important to not forget to note in the changelog the required update (in the case of using the hook)